danger.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'colored'
   spec.add_runtime_dependency 'nap'
   spec.add_runtime_dependency 'octokit'
+  spec.add_runtime_dependency 'rest-client'
```

Comment:
rest client is an extra dependency we don't need, faraday is already in the dependency graph, any chance you can use that? https://github.com/lostisland/faraday